android/app/src/main/AndroidManifest.xml

```diff
@@ -68,7 +68,6 @@
           <data android:scheme="https" android:host="rnbwapp.com" />
         </intent-filter>
       </activity>
-      <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
```

Comment:
Can we use the dev menu if we remove this?

Reply:
it is moved to another file